config/locales/es.yml

```diff
@@ -5902,7 +5897,7 @@ es:
         facts_html:
         - ¡Cobre su pago de estímulo de 2020 presentando su impuestos por medio de <a href="/">GetYourRefund</a>!
         - Revisa el estado de tu pago de estímulo en <a href='https://www.irs.gov/coronavirus/get-my-payment' data-track-click='stimulus-irs-get-my-payment' data-track-attribute-position='homepage' target='_blank' rel='noopener noreferrer'>IRS Obtener mi pago website</a>
-        - 'Llama a la Línea Directa de Pago de Impacto Económico 211 (en inglés: 211 Economic Impact Payment); <a href="tel:1-844-322-3639" data-track-click="call-211-hotline" data-track-attribute-position="homepage">+1 (844) 322-3639</a>'
+        - 'Llama a la Línea Directa de Pago de Impacto Económico 211 (en inglés: 211 Economic Impact Payment); <a href=''tel:1-844-322-3639'' data-track-click=''call-211-hotline'' data-track-attribute-position=''homepage''>+1 (844) 322-3639</a>'
```

Comment:
```suggestion
        - 'Llama a la Línea Directa de Pago de Impacto Económico 211 (en inglés: 211 Economic Impact Payment); <a href="tel:1-844-322-3639" data-track-click="call-211-hotline" data-track-attribute-position="homepage">+1 (844) 322-3639</a>'
```

Reply:
this keeps changing. it's copy for a ctc page that isn't in use anymore, so i've been ignoring it.